tools/homebrew-start/lanraragi

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+trap "kill 0" SIGINT
+
+# prefix/place where lrr is installed
+workdir=$(dirname $0)/$(dirname $(readlink $0))/..
+
+if [[ ${OSTYPE} == "darwin"* ]]; then
+  _LRR_DATA_DIRECTORY="${HOME}/Library/Application Support/LANraragi"
+else
+  _LRR_DATA_DIRECTORY="${HOME}/LANraragi"
+fi
+echo "Data Directory is set to '${_LRR_DATA_DIRECTORY}'"
+
+# modify redis config
+sed -i "" "s~<LRR_DATA_DIRECTORY>~${_LRR_DATA_DIRECTORY}~g" ${workdir}/redis.conf
```

Comment:
And one final GNU sed fix for the road.
```suggestion
sed -i"" "s~<LRR_DATA_DIRECTORY>~${_LRR_DATA_DIRECTORY}~g" ${workdir}/redis.conf
```
(see https://stackoverflow.com/questions/4247068/sed-command-with-i-option-failing-on-mac-but-works-on-linux)

Reply:
Might be better to have two commands for either mac and linux at this point looking at the comments below though. 👀 `-i""` doesn't seem to work all that well on mac.

Reply:
Yep, does not work on macOS. Neat, I love sed